src/server/routes/__tests__/forwardingRoute-test.js

```diff
@@ -109,6 +117,52 @@ test('forwardingRoute redirect with 301 if mapping OK (nn)', async () => {
   expect(next.notCalled).toBe(true);
 });
 
+test('forwardingRoute redirect learningpath with 301 if mapping OK (nb)', async () => {
+  prepareNock(200, '1337', 'urn:learningpath:122');
+
+  const next = sinon.spy();
```

Comment:
Is it possible to use https://jestjs.io/docs/en/jest-object#jestspyonobject-methodname ? 

It would be nice to remove sinon if possible?

Reply:
I don't think jest.spyOn supprots calledWith check?